src/cci/cci_handle_mng.c

```diff
@@ -139,7 +139,7 @@ static int
 compare_conn_info (unsigned char *ip_addr, int port, char *dbname, char *dbuser, char *dbpasswd, bool useSSL,
 		   T_CON_HANDLE * con_handle)
 {
-  if (con_handle->useSSL != useSSL ||  port != con_handle->port || memcmp (ip_addr, con_handle->ip_addr, 4) != 0
+  if (con_handle->useSSL != useSSL || port != con_handle->port || memcmp (ip_addr, con_handle->ip_addr, 4) != 0
```

Comment:
This line is not changed, it disturb me from concentration of what were changed, and what it is changed for.
I found several lines of this style. What about remove these lines which are not related with this PR.

Reply:
thanks for your comment